Artem Yurchenko/README.md

```diff
@@ -0,0 +1,56 @@
+
+## Products
+* [AIDA64 Extreme](https://www.aida64.com/products/aida64-extreme)
+* [AIDA64 Engineer](https://www.aida64.com/products/aida64-engineer)
+* [AIDA64 Business](https://www.aida64.com/products/aida64-business)
+* [AIDA64 Network Audits](https://www.aida64.com/products/aida64-network-audit)
+* [AIDA64 for Mobile devices](https://www.aida64.com/mobile-applications)
+
+
+## Updating to latest version
+In case you have a valid license, you can upgrade to a new stable or beta release, as long as your maintenance period has not yet expired. You can renew your maintenance period any time on our [License Renewal](https://www.aida64.com/support/renewal) page.
+You can learn more about AIDA64 licensing on our [Licensing page](https://www.aida64.com/licensing).
+
+
+## Frequently Asked Questions
+For the past nearly twenty years, we have received a large number of questions which have been asked time and again by our users. We have collected these, together with our answers, in our Knowledge Base, where you will find answers to the most common questions related to our products, potential technical issues, hardware and software support, licensing and payment methods.
+
+[Browse FAQ >](https://www.aida64.com/support/faq)
+
+
+## Why is my AIDA64 product key not working?
+AIDA64 product keys are uniquely generated based on the software edition, software version and other license parameters. The following cases are most common issues that prevent the software key from working properly.
+
+[Why is my AIDA64 product key not working? >](https://www.aida64.com/support/invalid-key)
+
+
+## Lost Product Key
+In the case you lost your product key and cannot activate AIDA64 Extreme, please follow the link below and enter your e-mail address you've used during the purchase.
+
+[Cleverbridge Purchase Lookup >](https://www.cleverbridge.com/750/?scope=cusecolp)
+
+If you using a AIDA64 Engineer Business, Network Audit or you cannot access your email address anymore or you were not able to obtain your product key using the link above, please submit a ticket.
+
+[Submit a ticket >](https://www.aida64.com/support/contactform)
+
+## Discussion Forums
+In the official AIDA64 forums, you can ask questions directly from the developers who are always ready to respond and help solve any AIDA64-related issues. Our products have also gathered together a community of enthusiasts and experts who are also glad to give a helping hand to users asking for support in the forums
+
+[Visit Forums >](https://forums.aida64.com/)
+
+## Have a question? Submit a ticket
```

Comment:
No need to put 'Submit a ticket' into the heading since it is in the description :) And also not only ticket submission can resolve an issue, as it is in the description again.